source/_integrations/zha.markdown

```diff
@@ -69,15 +69,17 @@ There is also support for Zigbee grouping of lights, switches, and fans (i.e. su
+Before installing the ZHA integration in Home Assistant, you will need to buy and connect a Zigbee Coordinator radio adapter or hardware module. Those normally come in the form of a USB dongle that plugs directly into the computer that is running the Home Assistant installation. The ZHA integration is compatible with many "Zigbee Coordinator" adapters, including Zigbee stacks (and Zigbee stack versions) from different manufacturers via their various serial interfaces. Be sure to [note the recommendations in the respective sections below before buying a new Zigbee radio adapter or hardware module](#compatible-hardware).
 
-In the case of ZHA, the Zigbee Coordinator is a radio adapter or hardware module, that contains a microcontroller that runs the Zigbee protocol stack which the ZHA integration uses to manage and communicate with a Zigbee network and its devices.
+In the case of this ZHA integration, the Zigbee Coordinator is a physical radio adapter or hardware module, that contains a microcontroller that runs the Zigbee stack which the ZHA integration uses to manage and communicate with the Zigbee network and all its devices. The Zigbee Coordinator will act as the central device in your Zigbee network. Its main responsibility from an end-user's point-of-view is the joining of devices to the network and then controlling them. Being the network bridge interface for Zigbee, it maintains the Zigbee network topology and ensures secure and efficient communication between Zigbee devices.
 
-Before installing the ZHA integration, you will need to buy and connect a Zigbee Coordinator radio adapter or hardware module, those usually come in the form of a USB dongle that plugs directly into the computer that is running the Home Assistant installation. The ZHA integration can work with many different "Zigbee Coordinator" adapters, however, be sure to read the respective sections below about compatible Zigbee radio adapters and hardware modules.
+Once ZHA has been set up and configured with a Zigbee Coordinator radio adapter or module you will be able to directly join/pair any Zigbee Router devices and Zigbee End Devices to the Zigbee network; [regardless of the manufacturer and brand of that Zigbee-based product](#knowing-which-devices-are-supported), (with few [limitations](#limitations)). While backward compatible with devices that use earlier versions of Zigbee Home Automation (ZHA) and Zigbee Light Link (ZLL) protocols, it is generally recommended to buy Zigbee 3.0 (ZB3) compliant devices. Zigbee 3.0 certified devices usually offer better interoperability with more Zigbee gateway solutions out-of-the-box, however [devices that use manufacturer-specific extensions to add non-standard functions and features will still need quirk code in custom device handlers to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).
```

Comment:
Terminology like "has been set up and configured" and "Zigbee Coordinator radio adapter or module" is unnecessarily verbose/redundant and hurts readability. Let's stick to simpler and shorter language, there's no need to be 100% precise in an introduction.

One possibility (roughly):

```suggestion
Once ZHA has been set up with a Zigbee radio, you will be able to join Zigbee devices to the network. While most devices will join, not all functionality [will always be supported out of the box](#knowing-which-devices-are-supported). It is generally recommended to buy Zigbee 3.0 devices. Zigbee 3.0 certified devices usually offer better interoperability, however devices that use manufacturer-specific extensions to add non-standard functions and features will still need [device-specific code to fully work with ZHA](#how-to-add-support-for-new-and-unsupported-devices).
```

Reply:
Updated to at least remove most jargon.

Reply:
Changed in https://github.com/home-assistant/home-assistant.io/pull/28869/commits/e9e308f0df0a4dc4eb1b27e706b27d7bfa7b7b1d